docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_creating_custom_test.adoc

```diff
@@ -91,6 +91,23 @@ spec:
 . Add your new custom test as an integration test in {ProductName}.
 .. For additional instructions on adding an integration test, see this document: xref:how-to-guides/testing_applications/proc_adding_an_integration_test.adoc[Adding an integration test].
 
+.Data injected into integration PipelineRun
+
+In the integration test created above, integration-service adds the following Parameters/Labels automatically to the integration PipelineRun.
+
+Parameters:
+
+ . `SNAPSHOT` containing the https://github.com/redhat-appstudio/application-api/blob/main/api/v1alpha1/snapshot_types.go[Snapshot] as a json string
+ . `NAMESPACE` containing the namespace where the snapshot will be deployed
```

Comment:
Can you clarify here that this parameter only gets added if the IntegrationTestScenario uses ephemeral environments to deploy the Snapshot that's being tested?

Reply:
Thanks. Added a NOTE for it.

Reply:
Thanks, looks good